gradle.properties

```diff
@@ -16,6 +16,7 @@ org.gradle.jvmargs=-Xmx3072m -XX:MaxMetaspaceSize=1536m -Dfile.encoding=UTF-8 -X
 org.gradle.parallel=true
 org.gradle.vfs.watch=true
 kotlin.code.style=official
+android.jetifier.ignorelist=jackson-core-2.16.1.jar
```

Comment:
CI tests are failing after updating the schemas to the latest version. Jackson-core is responsible for the failing tests. Hence, I found [this](https://issuetracker.google.com/issues/322403838#comment5) solution to ignore it from Jetifier.